docs/command-reference.md

```diff
@@ -464,6 +464,31 @@ Arguments:
 > Python: `def update(nef: bytes, manifest: str):`
 > 
 
+### neoxp contract download
+
+Download contract with storage from remote chain into local chain
+
+```
+Usage: neoxp contract download [Options] <Contract> <RpcUri>
+
+Arguments:
+  Contract              Target contract hash
+  RpcUri                URL of Neo JSON-RPC Node
+                        Specify MainNet (default), TestNet or JSON-RPC URL
+
+[Options]:
+  -i|--input <INPUT>    Path to neo-express data file
+  -h|--height <HEIGHT>  Block height to get contract state for
+                        Default value is: 0.
```

Comment:
we should explain that default means 'latest' (just like we did in the CLI help text https://github.com/neo-project/neo-express/commit/044200181278f50b2b19e838923904b0bc3b8720)